### ISSUE TITLE
Cannot install mikro-orm with node 14

### ISSUE THREAD

**Issue by abelland** [mikro-orm/mikro-orm#3212]
**Describe the bug**
Cannot install dependencies with node 14 as engine despite the requirement is node >= 14

**Stack trace**
```
error @side/jest-runtime@1.0.0: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.18.3"
error Found incompatible module.
```

**To Reproduce**
1. Download master branch
2. set node 14 as default engine
3. `yarn install`

**Expected behavior**
A clean installation of dependencies

**Additional context**
side/jest-runtime@1.0.0 requires [node >= 16](https://github.com/reside-eng/jest-runtime/blob/main/package.json#L11)

**Versions**

| Dependency | Version |
| - | - |
| node | 14.18.3 |
| typescript | 4.7.3 |
| mikro-orm | 5.2.0 |
| your-driver | N/A |

**Comment by B4nan** [comment]
We dont depend on jest, its a dev dependency 🤷

**Comment by B4nan** [comment]
Asked about the required version [here](https://github.com/reside-eng/jest-runtime/issues/14), let's see. But no plans to do anything about it on your end, I will rather tell devs to force install and have tests for node 18 than staying in the void (because of the jest memory leak, or better say v8 memory leak, we were not able to test on anything higher than node 16.10 without this plugin).

**Comment by abelland** [comment]
exactly, i mean locally. personally, i don't like to force the installation of packages but i get your point 👍
